### DISCUSSION TITLE
Works well but after how much time will it leave and will it start meetings.

### DISCUSSION THREAD

**Discussion by indrasen09** [TobiasPankner/Teams-Auto-Joiner#141] [Q&A]
So, I use leave_if_last: true. Will it only join a meeting or can It start a meeting(join button not present in the main screen). If the latter, does leave if last cause it to instantly quit ? Also is there any way to make it join a specific team and ignore others without having to blacklist all other teams?

**Comment by TobiasPankner** [comment]
The bot will never create a meeting, it only joins others.  
No there currently is no way of only joining one team, you could however hide all the other Teams in MS Teams.
